source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -67,6 +72,29 @@ export default class PinCode extends Component<Props> {
     }
   }
 
+  onKeyPress = (evt: SyntheticKeyboardEvent<EventTarget>, inputKey: string) => {
+    const { decimalSeparator, groupSeparator } = BigNumber.config().FORMAT;
+    const { key, target } = evt;
+    const control: { blur?: Function, focus?: Function } = target;
+    const focusKey = parseInt(inputKey, 10);
+    const nextFieldFocusKey = focusKey + 1;
+    const nextInputField = this.inputsRef[nextFieldFocusKey];
+    const isDotOrCommaKey = key === decimalSeparator || key === groupSeparator;
+    if (isDotOrCommaKey) {
```

Comment:
small issue but now it would be better if this was called `isSeparator` 😉

Reply:
Fixed